cmd/geth/main.go

```diff
@@ -417,10 +427,13 @@ func startNode(ctx *cli.Context, stack *node.Node, backend ethapi.Backend) {
 		// Set the gas price to the limits from the CLI and start mining
 		gasprice := utils.GlobalBig(ctx, utils.MinerGasPriceFlag.Name)
 		ethBackend.TxPool().SetGasPrice(gasprice)
-		// start mining
-		threads := ctx.GlobalInt(utils.MinerThreadsFlag.Name)
-		if err := ethBackend.StartMining(threads); err != nil {
-			utils.Fatalf("Failed to start mining: %v", err)
+		// SYSCOIN
```

Comment:
I'd add to the comment something about using a fake PoW with those flags, so we intentionally don't want to start mining. I know it sounds obvious, but just to be very clear.